src/lib/OpenEXRCore/openexr-c.c

```diff
@@ -0,0 +1,953 @@
+
+#define ILMTHREAD_THREADING_ENABLED
+#define OPENEXR_CORE_STANDALONE
+#define OPENEXR_VERSION_MAJOR 3 //@OpenEXR_VERSION_MAJOR@
+#define OPENEXR_VERSION_MINOR 2 //@OpenEXR_VERSION_MINOR@
+#define OPENEXR_VERSION_PATCH 0 //@OpenEXR_VERSION_PATCH@
```

Comment:
This seems very error-prone, so easy for us to bump a version (in the usual cmake way) but forget to change separately here. Is there a way to make both depend on a single source of truth so that the version is set in exactly one and only one place for the whole project?

Reply:
Addressed here. https://github.com/AcademySoftwareFoundation/openexr/pull/1386 When that lands, I'll update this PR to use that method instead.